openshift_tools/cloud/aws/ebs_snapshotter.py

```diff
@@ -396,9 +416,29 @@ def trim_snapshots(self, hourly_backups, daily_backups, weekly_backups, monthly_
                         errors.append(ex)
                 self.verbose_print()
 
-        self.verbose_print("Number of expired snapshots: %d" % len(all_expired_snapshots), prefix="  ")
-        self.verbose_print("Number of snapshots deleted: %d" % len(deleted_snapshots), prefix="  ")
-        self.verbose_print("Number of snapshot deletion errors: %d" % len(errors), prefix="  ")
+        # delete all orphaned snapshots older than N days
+        self.verbose_print("Removing orphaned snapshots older than {} days.\n".format(delete_orphans_older_than),
+                           prefix=" " * 2)
+
+        cutoff_date = datetime.utcnow() - timedelta(days=delete_orphans_older_than)
+        orphan_delete_counter = 0
+        for snapshot in orphan_snapshots:
+            snap_date = datetime.strptime(snapshot.start_time, '%Y-%m-%dT%H:%M:%S.000Z')
+            if snap_date < cutoff_date and 'snapshot' in snapshot.tags:
+                try:
+                    if dry_run:
+                        self.print_dry_run_msg(prefix=" " * 10, end=' Would delete {}\n'.format(snapshot.tags['Name']))
+                    else:
+                        snapshot.delete()
+                        orphan_delete_counter += 1
+                # pylint: disable=broad-except
```

Comment:
Please add a reason block here per our best practices:
https://github.com/openshift/openshift-tools/blob/prod/docs/best_practices_guide.adoc#All-PyLint-rule-disables-MUST-be-documented-in-the-code